src/components/coin-divider/CoinDivider.js

```diff
@@ -127,20 +117,6 @@ export default function CoinDivider({
     toggleOpenSmallBalances,
   } = useOpenSmallBalances();
 
-  useEffect(() => {
-    if (isSmallBalancesOpen && !fetchedCharts) {
-      const assetCodes = assets?.map(asset => asset.address);
-      dispatch(emitChartsRequest(assetCodes ?? []));
-      setFetchedCharts(true);
-    }
-  }, [
-    assets,
-    dispatch,
-    fetchedCharts,
-    isSmallBalancesOpen,
-    toggleOpenSmallBalances,
-  ]);
-
```

Comment:
cc @skylarbarrera it doesn't seem like anything was relying on this data fetch but CoinDivider is in the list a bunch, do you think there are any downstream effects here?

Reply:
smh, looks like just prefetching charts for a bunch of assets which we shouldn't do. fine to remove imo